README.md

```diff
@@ -2,14 +2,64 @@
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  1. Asymptotic analysis can be misleading for cases that use parallel
+     computing for higher performance. Synchronization and communication issues
+     are often present in these systems, which can heavily affect the runtime
+     of a program. Asymptotic analysis analyzes the work of an algorithm, but 
+     it cannot predict the issues that come with multiple processors communicating
```

Comment:
We can analyze communication complexity (although we don't do that in this class).